addons/common/functions/fnc_unloadPersonLocal.sqf

```diff
@@ -20,11 +20,13 @@
 #define GROUP_SWITCH_ID QFUNC(loadPerson)
 
 params ["_unit", "_vehicle", ["_unloader", objNull]];
-TRACE_3("params",_unit,_vehicle,_unloader);
+TRACE_3("unloadpersonLocal",_unit,_vehicle,_unloader);
 
 //This covers testing vehicle stability and finding a safe position
 private _emptyPos = [_vehicle, (typeOf _unit), _unloader] call EFUNC(common,findUnloadPosition);
 TRACE_1("findUnloadPosition",_emptyPos);
+if (_emptyPos isEqualTo []) then { _emptyPos = [_vehicle, (typeOf _unit), _unloader] call EFUNC(common,findUnloadPosition); TRACE_1("findUnloadPosition 2nd attempt",_emptyPos); };
+if (_emptyPos isEqualTo []) then { _emptyPos = [_vehicle, (typeOf _unit), _unloader] call EFUNC(common,findUnloadPosition); TRACE_1("findUnloadPosition 3rd attempt",_emptyPos); };
```

Comment:
Duplicated line

Reply:
Seems to be intended.

Reply:
Yes, missed the "3rd attempt" part.

Reply:
probably because line was too wide  :smile: